README.org

```diff
@@ -18,3 +18,6 @@ Here's a table describing the bindings
 | =ediff-suspend=             | =z=              | =C-z=          |
 | scroll left                 | =>=              | =zh=           |
 | scroll right                | =<=              | =zl=           |
+| copy A region to B's region | a                | l              |
+| copy B region to A's region | b                | h              |
```

Comment:
yes, naturally, h <- and l ->

indeed for the vertial split is not so intuitive anymore,
but I my opinion still worth it

btw. i find the verstil split weird, and never use it,
 as I am used to BeyondCompare which alignes the matched lines
 between the files on the same height, I see that ediff doesn't do that,
I am not sure, if that feature is missing in ediff or it's just my settings